boilerplate/app/screens/DemoShowroomScreen/DemoShowroomScreen.tsx

```diff
@@ -177,24 +187,28 @@ export const DemoShowroomScreen: FC<DemoTabScreenProps<"DemoShowroom">> =
 
           <SectionList
             ref={listRef}
-            contentContainerStyle={$sectionListContentContainer}
+            contentContainerStyle={themed($sectionListContentContainer)}
             stickySectionHeadersEnabled={false}
-            sections={Object.values(Demos)}
+            keyExtractor={(item, index) => item.name + index}
```

Comment:
I don't think this is working as intended for me. I'm getting a unique key warning, and when I log the values it's giving undefined for `item.name` and 0 for index. Which is very weird.